docs/user-manual/en/amqp-server-connections.md

```diff
@@ -0,0 +1,230 @@
+        <peer match="queues.#"/>
+    </amqp-connection>
+</broker-connections>
+```
+*Important:* The target endpoint needs permission on all operations you configure. So if you're using a security manager make sure you use an user with enough privileges to perform your configured operations.
+
+# AMQP Server Connection elements
+We currently support the following types of elements on a AMQP Server Connection:
+
+- Senders
+    - Messages received on specific queues will be transferred to another endpoint
+- Receivers
+    - The broker will pull messages from another endpoint
+- Peers
+    - The broker will create both senders and receivers to another endpoint that knows how to handle it. This is currently implement by qpid-dispatch.
+- Mirrors
+    - The broker will use an AMQP connection towards another broker and duplicate messages and send acknowledgements over the wire.
+     
+This is a snippet on how you configure an AMQP Server Connection:
+
+```xml
+<broker-connections>
+ <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker">
+     ... amqp lements ...
+ </amqp-connection>
+</broker-connections>
+```
+
+## Senders and Receivers
+It is possible to connect an ActiveMQ Artemis Broker towards another AMQP endpoint, by simply creating a sender or receiver broker connection element.
+
+For a `sender`, the broker will create a message consumer on a queue, that will *send* messages towards another AMQP endpoint.
+
+For a `receiver`, the broker will create a message producer on an address, that will *receive* messages from another AMQP endpoint.
+
+Both elements will work like a message bridge however without any extra weight required to process messages. They are just like any other consumer or producer as part of ActiveMQ Artemis Broker.
+
+Senders and Receivers can be specified by address matching using [wildcard expressions](wildcard-syntax.md).
+
+There are two properties you can use to configure a sender or receiver:
+
+- match: for the wildcard expression mathing addresses
+- queue-name: The name of a queue, without expressions
+
+Example of configuring senders and receivers using address expressions:
+
+```xml
+<broker-connections>
+ <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker">
+    <sender match="queues.#"/>
+    <!-- notice the local queues for removequeues.# need to be created on this broker -->
+    <sender match="remotequeues.#"/>
+ </amqp-connection>
+</broker-connections>
+
+<addresses>
+ <address name="remotequeues.A">
+    <anycast>
+       <queue name="remoteQueueA"/>
+    </anycast>
+ </address>
+ <address name="queues.B">
+    <anycast>
+       <queue name="localQueueB"/>
+    </anycast>
+ </address>
+</addresses>
+```
+
+Example of configurig senders and receivers using queue names:
+
+```xml
+<broker-connections>
+    <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker">
+        <sender queue-name="remoteQueueA"/>
+        <sender queue-name="localQueueB"/>
```

Comment:
As above, is one meant to be a receiver?

Reply:
Yeah.. thanks.. marking this conversation as resolved